CenterForOpenScience/osf.io
# [ENG-4148] Add tab to legacy UI to go to project metadata page

## Purpose

The purpose of these changes is to add a Metadata tab to the Legacy UI.

## Changes

The current pattern for tabs was followed for adding a Metadata tab to the Projects/Components page of the OSF.

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify tab exists
- Verify tab contains proper label for Metadata
- Verify tab properly redirects the user to the file detail page

What are the areas of risk?

Improper redirection, non-functional button tab.

## Documentation

Documentation/Mock-ups provided here: 
https://preview.uxpin.com/c18fdf7758289c72cd8a7d77b71dccf5cddb4ad7#/pages/159623837/simulate/sitemap?mode=i

## Side Effects

Spillage/crowding of the navigation menu 

## Ticket

[](https://openscience.atlassian.net/browse/ENG-4148)
